src/dialogs.c

```diff
@@ -460,27 +461,20 @@ void dialogs_show_open_file(void)
 
 	SETPTR(initdir, utils_get_locale_from_utf8(initdir));
 
-#ifdef G_OS_WIN32
-	if (interface_prefs.use_native_windows_dialogs)
-		win32_show_document_open_dialog(GTK_WINDOW(main_widgets.window), _("Open File"), initdir);
-	else
-#endif
-	{
-		GtkWidget *dialog = create_open_file_dialog();
+	dialog = create_open_file_dialog();
+	open_file_dialog_apply_settings(dialog);
 
-		open_file_dialog_apply_settings(dialog);
+	if (initdir != NULL && g_path_is_absolute(initdir))
+			gtk_file_chooser_set_current_folder(GTK_FILE_CHOOSER(dialog), initdir);
 
-		if (initdir != NULL && g_path_is_absolute(initdir))
-				gtk_file_chooser_set_current_folder(GTK_FILE_CHOOSER(dialog), initdir);
+	if (app->project && !EMPTY(app->project->base_path))
+		gtk_file_chooser_add_shortcut_folder(GTK_FILE_CHOOSER(dialog),
+				app->project->base_path, NULL);
 
-		if (app->project && !EMPTY(app->project->base_path))
```

Comment:
Why is adding this shortcut removed?

Reply:
Bah!! ignore comment, githubs annoying handling of changed indent confused me.

Reply:
Yeah, sorry. The diffs are bit hard to read because of indentation changes :(: